### PR TITLE
Biosys 277

### PR DESCRIPTION
**Pull request by chungvl**
Fabs hide when they are present on a "no-data-onboarding" page.

### CODE REVIEW

File: src/components/records-list/records-list.ts [hunk 1]
```diff
@@ -1,10 +1,14 @@
 import { Component, Input } from '@angular/core';
-import { NavController, NavParams } from 'ionic-angular';
+import { Events, NavController, NavParams, ToastController } from 'ionic-angular';
```
Comment (tony-gaia) [inline src/components/records-list/records-list.ts]: Is the ToastController being used?

File: src/pages/home/home.html [hunk 2]
```diff
@@ -10,12 +10,12 @@
 <ion-content>
     <ion-tabs tabsPlacement="top">
         <ion-tab tabTitle="List" title="List"
-                 [rootParams]="{showLegend: true, data: records, navCtrl: navCtrl, haveTree: false}"
+                 [rootParams]="{showLegend: true,data: records,navCtrl: navCtrl,haveTree: false,haveUpload: true}"
```
Comment (tony-gaia) [inline src/pages/home/home.html]: keep the spaces

Comment (tony-gaia) [inline src/pages/home/home.html]: keep the spaces

File: src/components/records-list/records-list.ts [hunk 3]
```diff
@@ -40,11 +44,17 @@ export class RecordsListComponent {
     @Input()
     public haveTree = true;
 
+    @Input()
+    public haveUpload = false;
+
     @Input()
     public haveWelcome = true;
+    private countMap = {};
```
Comment (tony-gaia) [inline src/components/records-list/records-list.ts]: Doesn't look like this is used

**Comment by chungvl** [comment]
All fixed.

**Pull request merged**